### PR TITLE
added new functions to c-api

### PR DESCRIPTION
**Pull request by abrosimov**
Few functions from options added to C-api

### CODE REVIEW

File: include/rocksdb/c.h [hunk 1]
```diff
@@ -1765,6 +1771,10 @@ extern ROCKSDB_LIBRARY_API uint64_t
 rocksdb_approximate_memory_usage_get_cache_total(
     rocksdb_memory_usage_t* memory_usage);
 
+extern ROCKSDB_LIBRARY_API void rocksdb_options_set_dump_malloc_stats(rocksdb_options_t*, int);
```
Comment (ltamasi) [inline include/rocksdb/c.h]: Could you change the type of the second parameter to `unsigned char`? That's what we generally use as a stand-in for `bool` in the C API. Note that this applies to the next method as well.

**Comment by ltamasi** [comment]
Thanks @abrosimov ! Could you please rebase your patch and run the code formatter (`make format`)?

File: include/rocksdb/c.h (hunk 1 above)
Comment (abrosimov) [reply include/rocksdb/c.h]: done

**Comment by abrosimov** [comment]
@ltamasi I did it, but there is some problems with CI (travic-ci tests took more than 50 minutes).
I think that these problems doesn't related to changes I've made

**Pull request closed without merging**